Tiltfile

```diff
@@ -166,18 +228,31 @@ def deploy_cspo():
 
     # Set up an image build for the provider. The live update configuration syncs the output from the local_resource
     # build into the container.
-    docker_build_with_restart(
-        ref = "ghcr.io/sovereigncloudstack/cspo-staging",
-        context = "./.tiltbuild/",
-        dockerfile_contents = tilt_dockerfile_header_cspo,
-        target = "tilt",
-        entrypoint = entrypoint,
-        only = "manager",
-        live_update = [
-            sync(".tiltbuild/manager", "/manager"),
-        ],
-        ignore = ["templates"],
-    )
+    if settings.get("local_mode"):
+        docker_build_with_restart(
+            ref = "ghcr.io/sovereigncloudstack/cspo-staging",
+            context = ".",
+            dockerfile_contents = tilt_dockerfile_header_cspo,
+            target = "tilt",
+            entrypoint = entrypoint,
+            live_update = [
+                sync(".tiltbuild/manager", "/manager"),
+                sync(".release", "/tmp/downloads/cluster-stacks"),
+            ],
+            ignore = ["templates"],
+        )
+    else:
+        docker_build_with_restart(
+            ref = "ghcr.io/sovereigncloudstack/cspo-staging",
+            context = "./.tiltbuild/",
+            dockerfile_contents = tilt_dockerfile_header_cspo,
+            target = "tilt",
+            entrypoint = entrypoint,
+            live_update = [
+                sync(".tiltbuild/manager", "/manager"),
+            ],
+            ignore = ["templates"],
+        )
     k8s_yaml(blob(yaml))
     k8s_resource(workload = "cspo-controller-manager", labels = ["cspo"])
     k8s_resource(
```

Comment:
I suggest to add to objects here all the uncategorized objects, e.g. `openstackclusterstackreleases.infrastructure.clusterstack.x-k8s.io:customresourcedefinition`...